File: solutions/python/raindrops/1/raindrops.py
"""This is the raindrop exercism project"""
def convert(number):
    """This function makes pling plang plong depending on divisibility"""
    response = ""
    if number % 3 == 0:
        response += "Pling"

    if number % 5 == 0:
        response += "Plang"

    if number % 7 == 0:
        response += "Plong"
    
    if not response:
        return str(number)
    
    return response

File: solutions/cpp/raindrops/2/raindrops.cpp
#include "raindrops.h"
#include <string>
namespace raindrops {
    
// TODO: add your solution here
    std::string convert(int number){
        std::string result = "";
        
        if(number%3==0){
            result += "Pling";
        }
        
        if(number%5==0){
            result += "Plang";
        }
        
        if(number%7==0){
            result += "Plong";
        }

        else{
            result+= std::to_string(number);
        }

        return result;
    }

}  // namespace raindrops
